### PR TITLE
BloClick BP-03-001

### PR DESCRIPTION
**Pull request by ivandiazperez**

### CODE REVIEW

File: testnet/tasks/BP-01-001/lEcF3EcVRmrV.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+    "uid": "lEcF3EcVRmrV",
+    "api_key_name": "bloclick"
+}
```
Comment (CloverRL) [inline testnet/tasks/BP-01-001/lEcF3EcVRmrV.json:4]: re-01-001

Comment (CloverRL) [reply testnet/tasks/BP-01-001/lEcF3EcVRmrV.json:4]: re-02-002

File: testnet/tasks/BP-01-003/lEcF3EcVRmrV.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+    "uid": "lEcF3EcVRmrV"
+}
```
Comment (CloverRL) [inline testnet/tasks/BP-01-003/lEcF3EcVRmrV.json:3]: re-01-003

File: testnet/tasks/BP-01-001/lEcF3EcVRmrV.json (hunk 1 above)
Comment (0xcoco) [inline testnet/tasks/BP-01-001/lEcF3EcVRmrV.json:4]: reviewed 01-002

**Comment by Mummyhap** [comment]
reviewed 03-001

**Comment by 0xcoco** [comment]
reviewed 02-001

**Pull request closed without merging**